packages/indiekit/config/express.js

```diff
@@ -10,12 +11,15 @@ import { logging } from "../lib/middleware/logging.js";
 import { routes } from "../lib/routes.js";
 import { views } from "../lib/views.js";
 
+const debug = makeDebug(`indiekit:express`);
+
 /**
  * @typedef {import("express").Application} Application
  * @param {object} indiekitConfig - Indiekit configuration
  * @returns {Application} Express application
  */
 export const expressConfig = (indiekitConfig) => {
+  debug(`create Express app`);
```

Comment:
Has having these separate debug outputs helped you? Has the app failed to set up between these different steps, for example?

Reply:
I tend to add debug logs in key points of the server creation, like when I add an Express middleware or a Fastify plugin.

For example, if there is a `debug` call before I register middleware A and middleware B, and I see only the log statement from middleware A, I know the app crashed between A and B, or that I misconfigured middleware B.

However, I think these debug logs are way less important than the ones I added to troubleshoot the connection to Mongo DB. I view the debug logs in this module as a _nice to have_, not a requirement.